src/dune/expander.ml

```diff
@@ -328,6 +331,8 @@ let parse_lib_file ~loc s =
 let cc t ~dir =
   let open Build.O in
   let cc = t.foreign_flags ~dir in
+  let cflags = Build.map cc.c ~f:(fun fl -> t.cflags @ fl) in
```

Comment:
Why don't we do this for cxxflags as well?

Reply:
Because for cxx flags it is already done in `Super_context.default_context_flags`.

It's not the case anymore for the cflags as it was discussed in #3565 (making `:standard` empty for cflags).

However we could regain symetry by adding back the flags in `default_contewt_flags` and not use that for `:standard`.

Reply:
I see. Seems like keeping it symmetric is better? Otherwise it will be harder to switch to the new behavior when for C/Cxx.

Reply:
I just had a look and I am not sure it makes sense anymore:

`SC.default_context_flags` is merged as `:standard` with flags from the `env` stanza in `Env_node.make`.
And these are merged as `:standard` with flags from the various `foreign_stubs` stanzas.

There is no other use of `SC.default_context_flags` , it relaly is the "`:standard`" set of flags and if we want this set of flags to be empty for C stubs then I think we should keep the C part of 
`SC.default_context_flags` empty.